tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -79,6 +79,10 @@ class Settings(Base):
 
     _main_title_locator = (By.ID, 'main-list-title')
 
+    def __init__(self, marionette):
+        Base.__init__(self, marionette)
+        self._airplane_checkbox = HtmlBinaryControl(self.marionette, self._airplane_checkbox_locator)
+
```

Comment:
The __init__ things you added are causing all the failures on Jenkins, it seems.
The failures happen because when a Settings instance is called, for instance, it doesn't have to be launched yet.